cypress/integration/networking.spec.ts

```diff
@@ -92,6 +99,9 @@ context("p2p networking", () => {
                 throw new Error("Could not find URN");
               }
 
+              // We’re giving the P2P node some space to avoid race
+              // conditions.
+              cy.sleep(2000);
```

Comment:
I think the sleeps are redundant, because we already set `defaultCommandTimeout: 8000` to 8s. Maybe we should double that to 16s?

Reply:
The `sleep` calls are different from the timeout. The timeout controls how long commands are retried. But the first try is issued immediately. The purpose of `sleep()` is to delay the next command (like pushing changes with git) so that the nodes have time to settle down.

Reply:
Ah, that makes sense.